roles/openshift-prep/tasks/prerequisites.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: "Cleaning yum repositories"
+  command: "yum clean all"
+  
+- name: "Install required packages"
+  yum:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - wget
+    - git
+    - net-tools
+    - bind-utils
+    - bridge-utils
+    - bash-completion
+    - atomic-openshift-utils
```

Comment:
to clarify, this package limits the use cases for the provider to the OpenShift enterprise only. Please let's assure it's working with origin on centos7 as well. Perhaps we could you this docker images for the upstream origin deployment cases https://hub.docker.com/r/karafior/atomic-openshift-utils/  ?